explorer/e2e/testFunctions.ts

```diff
@@ -1,8 +1,22 @@
-import { expect, Page } from "@playwright/test";
+import { expect, Locator, Page, test } from "@playwright/test";
 import { TabDescription } from "./testInterfaces";
 
 /* eslint-disable sonarjs/no-duplicate-string  -- ignoring duplicate strings here */
 // Run the "Expect each tab to appear as selected when the corresponding url is accessed" test
+
+export const getFirstElementTextLocator = (
```

Comment:
Would you be able to add [TSDocs](https://tsdoc.org/) for each function in this functions file? You can see examples of what I'm expecting [here](https://github.com/DataBiosphere/data-browser/blob/main/explorer/files/common/utils.ts#L15), [here](https://github.com/DataBiosphere/data-browser/blob/main/explorer/files/common/utils.ts#L25), or [here](https://github.com/DataBiosphere/data-browser/blob/main/explorer/app/utils/tsvParser.ts#L15).